docs/notebooks/ppg/ppg_analysis.ipynb

```diff
@@ -56,14 +52,14 @@
     "imu_config = IMUConfig()\n",
     "metadatas_ppg, metadatas_imu = scan_and_sync_segments(os.path.join(path_to_sensor_data, 'ppg'),\n",
     "                                                       os.path.join(path_to_sensor_data, 'imu'))\n",
-    "df_ppg_proc, df_imu_proc=preprocess_ppg_data(metadatas_ppg[0], metadatas_imu[0],\n",
+    "df_ppg_proc, df_acc_proc = preprocess_ppg_data(metadatas_ppg[0], metadatas_imu[0],\n",
```

Comment:
The indentation is a little off in this cell block. It's difficult to provide suggestions, but in general for longer lines of code using functions, try entering a new line after the opening bracket "(" and a final line for the closing bracket ")".

Reply:
Agree, I have created now a single line again since it is also not that long